holmes/plugins/toolsets/prometheus/prometheus.py

```diff
@@ -522,6 +522,11 @@ def __init__(self, toolset: "PrometheusToolset"):
                     type="number",
                     required=True,
                 ),
+                "output_type": ToolParameter(
+                    description="The output type is case-sensitive and must be one of the following: Plain, Bytes, Percentage, or CPUUsage.",
+                    type="string",
+                    required=True,
+                ),
```

Comment:
the param description does not seem to instruct holmes on how to set this argument. What is it for?